frontend/src/components/ContactUs.tsx

```diff
@@ -0,0 +1,165 @@
+import { createMuiTheme, ThemeProvider } from '@material-ui/core/styles'
+
+
+const useStyles = makeStyles({
+    root:{
+        '& .MuiFormControl-root':{
+            width: '80%',
+            margin: 8,
+        },
+      
+    },
+    pageSideBar:{
+        margin: 8,
+        padding: 50,
+        backgroundColor: "rgb(238,235,228)",
+        
+    },
+   
+    pageContent:{
+        margin: 8,
+        padding: 110,
+        backgroundColor: "rgb(234,232,224)",
+        
+    },
+    submitButton:{
+        margin: 1,
+        
+    },
+    cancelButton:{
+        margin: 1,
+
+    },
+    sideBar:{
+        fontSize: '1rem',
+        fontWeight: 'bold',
+        cursor: 'pointer',
+        '@media (min-width:1100px)':{
+            fontSize: '1.5rem',
+        },
+
+    },
+
+})
+
+const contactValue = {
+    name: '',
+    email: '',
+    message: '',
+
+}
+
+export default function ContactUs() {
+    const [values, setValues] = useState(contactValue);
+    const classes = useStyles();
+
+    const getInput= e =>{
+        const {name, value } = e.target
+        setValues({
+            ... values,
+            [name]: value
+        })
+    }
+
+    return (
+        <>
+       
+        <Grid container>
+            <Paper className={classes.pageSideBar}>
+                <Grid container spacing={2} direction="column" justify="center">
+                    <Grid item>
+                        <Typography className={classes.sideBar}>
+                            Report a Video
+                        </Typography>
+                    </Grid>
+                    <Grid item>
+                        <Typography className={classes.sideBar}>
+                            Report an Issue
+                        </Typography>
+                    </Grid>
+                    <Grid item>
+                        <Typography className={classes.sideBar}>
+                            Provide Feedback
+                        </Typography>
+                    </Grid>
+                    <Grid item>
+                        <Typography className={classes.sideBar}>
+                            Report a Profile
+                        </Typography>
+                    </Grid>
+                    <Grid item>
+                        <Typography className={classes.sideBar}>
+                            FAQs
+                        </Typography>
+                    </Grid>
+
+                </Grid>
+
+            </Paper>
+
+
+
+                <Paper className={classes.pageContent}>
+                    <form className={classes.root}>
+                        <Grid container spacing={2} direction="column" justify="center">
+                            <Grid item xs={12}>
+                                <TextField
+                                variant="outlined"
+                                label="Your name"
+                                name = "name"
+                                value={values.name}
```

Comment:
What exactly does setting `value` do here? I think I remember having issues in the Profile Editing branch when I set the value attribute because it wouldn't let me edit the value once this attribute is set.

Reply:
Setting the value here is to get the information from the user.  The information then would be used to update the contact value when the getInput function is called.